salt/modules/dockerng.py

```diff
@@ -926,8 +932,34 @@ def _image_wrapper(attr, *args, **kwargs):
     catch_api_errors = kwargs.pop('catch_api_errors', True)
 
     if kwargs.pop('client_auth', False):
-        # Set credentials
-        registry_auth_config = __pillar__.get('docker-registries', {})
+        # Get credential from the home directory of the user running
+        # salt-minion, default auth file for docker (~/.docker/config.json)
+        registry_auth_config = {}
+        try:
+            home = os.path.expanduser("~")
+            docker_auth_file = os.path.join(home, '.docker', 'config.json')
+            if os.path.isfile(docker_auth_file):
```

Comment:
The method of error catching here is a bit clunky. The more pythonic way would be to simply try to read the file.

```python
with salt.utils.fopen(docker_auth_file) as fp:
    try:
        docker_auth = json.load(fp)
    except (OSError, IOError) as exc:
        if exc.errno != errno.ENOENT:
            log.error('Failed to read docker auth file %s: %s', docker_auth_file, exc)
        docker_auth = {}